samples/nextjs/src/pages/_edit.tsx

```diff
@@ -1,14 +1,13 @@
 import { NextPageContext } from 'next';
-import Error from 'next/error';
+import NotFound from 'components/NotFound';
 import Layout from 'components/Layout';
 import { SitecoreContext, EditingRequest } from '@sitecore-jss/sitecore-jss-nextjs';
 import { SitecorePageProps } from 'lib/page-props';
 import { componentFactory } from 'temp/componentFactory';
 
 const SitecorePage = ({ layoutData }: SitecorePageProps): JSX.Element => {
   if (!layoutData?.sitecore?.route) {
-    // layoutData will be missing for an invalid path
-    return <Error statusCode={404} />;
+    return <NotFound />;
```

Comment:
@ambrauer Can you provide `context` here?
`<NotFound context={layoutData?.sitecore.context} />;`

Reply:
@sc-illiakovalenko Added. I didn't add originally because (at least at the present) in EE, we would never have context without route.